tests/zone_tests.cpp

```diff
@@ -163,7 +163,7 @@ TEST_CASE("Zones from 3 Wavs", "[zones]")
             else
             {
                 INFO("Checking with note " << n);
-                std::vector<float> vals = {16.963544, 14.8987890705, 8.9075824};
+                std::vector<float> vals = {16.964918, 14.899617, 8.906973};
```

Comment:
These ones are fine - it’s just missing one block i bet.

Reply:
Yea this one looked trivial. Perhaps a bit of fluctuation but from what I can tell, it's still achieving the intent.